index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>The Portfolio</title>
+    <script src="https://kit.fontawesome.com/0ec53d8e70.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="/dist/style.css">
+</head>
+<body>
+    <header class="navigation">
+      <div class="logo">
+        <h2>Evans Karanja</h2>
```

Comment:
- [ ] Kindly make sure to wrap the logo into an ancho tag `a` because of three reasons: convention, semantics, and user experience(UX). [Check this link](https://ux.stackexchange.com/questions/81727/why-is-it-standard-for-a-website-logo-to-navigate-to-the-home-page) for more info.